echidna/scripts/dump_smeared.py

```diff
@@ -49,11 +51,20 @@
     if args.smear_method == "weight":  # Use default smear method
         smeared_spectrum = smearer.weight_gaussian_energy_spectra(spectrum)
         smeared_spectrum = smearer.weight_gaussian_radius_spectra(smeared_spectrum)
+    elif args.smear_method == "weight_parallel":  # Use default smear method
```

Comment:
Could you add these new smear methods to the note on valid smear methods in the module docstring please.